webpack.config.js

```diff
@@ -12,10 +13,21 @@ module.exports = {
     },
     module: {
         loaders: [
-            { test: /\.css$/,  loader: 'style!css' },
+            {
+                test:   /\.css$/,
+                loader: "style!css!postcss-loader"
```

Comment:
No. Dangling commas are used for objects and arrays members, not for statements.